pydantic_settings/main.py

```diff
@@ -18,7 +15,7 @@
     SecretsSettingsSource,
 )
 
-env_file_sentinel = str(object())
+env_file_sentinel: DotenvType = FilePath('')
```

Comment:
What motivated this change? Just curious

Reply:
make mypy happy